examples/3d/volumetric_fog.rs

```diff
@@ -60,6 +81,37 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             ..default()
         });
 
+    // Add the point light
+    commands
+        .spawn(PointLightBundle {
```

Comment:
Would be nice to be able to move the point light around. Just moving statically between back and forth between the walls would probably be enough. Not really a blocker for the PR, just something that would be nice.

Reply:
The point light now moves back and forth between the left and right walls.

https://github.com/user-attachments/assets/3eeb77a0-f22d-40a6-a48a-2dd75d55a877